streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -2491,26 +2467,17 @@ public Set<TopicPartition> partitions() {
             "K2".getBytes(),
             "V2".getBytes()));
 
-        if (stateUpdaterEnabled) {
-            TestUtils.waitForCondition(
+        TestUtils.waitForCondition(
                 () -> mockRestoreConsumer.assignment().size() == 0,
                 "Never get the assignment");
```

Comment:
Fix indentation and could you please also fix the typo? It should be `Never got the assignment`.